items/src/test/scala/weco/api/items/services/ItemUpdateServiceTest.scala

```diff
@@ -0,0 +1,131 @@
+    with JsonAssertions
+    with ScalaFutures
+    with ItemsApiGenerators {
+
+  val sierraItemNumber = createSierraItemNumber
+  val dummyDigitalItem = createDigitalItem
+
+  val availableItemResponses = Seq(
+    (
+      HttpRequest(uri = sierraUri(sierraItemNumber)),
+      HttpResponse(
+        entity = sierraItemResponse(
+          sierraItemNumber = sierraItemNumber
+        )
+      )
+    )
+  )
+
+  val onHoldItemResponses = Seq(
+    (
+      HttpRequest(uri = sierraUri(sierraItemNumber)),
+      HttpResponse(
+        entity = sierraItemResponse(
+          sierraItemNumber = sierraItemNumber,
+          holdCount = 1
+        )
+      )
+    )
+  )
+
+  val temporarilyUnavailableItem: Item[IdState.Identified] = {
+    val temporarilyUnavailableOnline = AccessCondition(
+      method = AccessMethod.OnlineRequest,
+      status = AccessStatus.TemporarilyUnavailable
+    )
+
+    createPhysicalItemWith(
+      sierraItemNumber = sierraItemNumber,
+      accessCondition = temporarilyUnavailableOnline
+    )
+  }
+
+  val availableItem = {
+    val temporarilyUnavailableOnline = AccessCondition(
+      method = AccessMethod.OnlineRequest,
+      status = AccessStatus.TemporarilyUnavailable
+    )
+
+    createPhysicalItemWith(
+      sierraItemNumber = sierraItemNumber,
+      accessCondition = temporarilyUnavailableOnline
+    )
+  }
+
+  val onHoldAccessCondition = AccessCondition(
+    method = AccessMethod.ManualRequest,
+    status = Some(TemporarilyUnavailable),
+    note = Some("Item is in use by another reader. Please ask at Enquiry Desk.")
+  )
+
+  val onlineRequestAccessCondition = AccessCondition(
+    method = AccessMethod.OnlineRequest
+  )
+
+  val workWithUnavailableItem = indexedWork().items(
+    List(
+      temporarilyUnavailableItem,
+      dummyDigitalItem
+    )
+  )
+
+  val workWithAvailableItem = indexedWork().items(
+    List(
+      availableItem,
+      dummyDigitalItem
+    )
+  )
+
+  val itemStates = Table(
+    ("Sierra Responses", "Catalogue Work", "AccessCondition"),
+    (onHoldItemResponses, workWithAvailableItem, onHoldAccessCondition),
+    (onHoldItemResponses, workWithUnavailableItem, onHoldAccessCondition),
+    (
+      availableItemResponses,
+      workWithAvailableItem,
+      onlineRequestAccessCondition
+    ),
+    (
+      availableItemResponses,
+      workWithUnavailableItem,
+      onlineRequestAccessCondition
+    )
+  )
+
+  forAll(itemStates) {
+    (sierraResponses, catalogueWork, expectedAccessCondition) =>
+      withItemUpdateService(sierraResponses) { itemUpdateService =>
+        whenReady(itemUpdateService.updateItems(catalogueWork)) {
+          updatedItems =>
+            updatedItems.length shouldBe 2
+
+            val physicalItem = updatedItems(0)
+            val digitalItem = updatedItems(1)
+
+            physicalItem.locations.head.accessConditions.head shouldBe expectedAccessCondition
+            digitalItem shouldBe dummyDigitalItem
+        }
+      }
+  }
```

Comment:
Does this need to be wrapped in an `it` block or something?

Reply:
Weirdly it works if you don't - but it gives slightly nicer output if you do.